################################################################################
SHORT DESCRIPTION: 
################################################################################
Use the first fb driver loaded instead of the primary device while using DRM
KMS helper.

################################################################################
LONG DESCRIPTION: 
################################################################################
Remove auto-select FRAMEBUFFER_CONSOLE_DETECT_PRIMARY from DRM_KMS_FB_HELPER.

################################################################################
CHANGELOG 
################################################################################
Original Author: unknown
Port to 3.18: Eric Chanudet <chanudete@ainfosec.com>
Port to 4.14: Richard Turner <turnerr@ainfosec.com>
Port to 4.19: Richard Turner <turnerr@ainfosec.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
None, this is an OpenXT work-around.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################
Is required by surfman/drm-plugin.

################################################################################
PATCHES 
################################################################################
--- a/drivers/gpu/drm/Kconfig
+++ b/drivers/gpu/drm/Kconfig
@@ -81,7 +81,6 @@ config DRM_KMS_FB_HELPER
 	depends on DRM_KMS_HELPER
 	select FB
 	select FRAMEBUFFER_CONSOLE if !EXPERT
-	select FRAMEBUFFER_CONSOLE_DETECT_PRIMARY if FRAMEBUFFER_CONSOLE
 	select FB_SYS_FOPS
 	select FB_SYS_FILLRECT
 	select FB_SYS_COPYAREA
